notebooks/intercomparison-covermaps.ipynb

```diff
@@ -0,0 +1,1346 @@
+{
```

Comment:
<ol><li><code>dir</code> is also a python command so I suggest not using it as a variable</li><li><code>Path</code> from the <code>pathlib</code> library, once you get a hang of it, it's a lot cleaner than using strings for paths</li></ol><pre class="ql-syntax" spellcheck="false">from openmapflow.config import PROJECT_ROOT, DataPaths # This uses pathlib behind the scenes

all_dataset_paths = (PROJECT_ROOT / DataPaths.DATASETS).glob("*")
test_paths = [p for p in all_dataset_paths if p.stem in ["Kenya", "Togo"]

...

for p in test_paths:
   df = pd.read_csv(p)
   df["country"] = p.stem
</pre><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>